src/coral_models/prepare_raw_data.py

```diff
@@ -0,0 +1,470 @@
+"""Functions for preparing the raw data"""
+
+import contextlib
+import datetime
+import sqlite3
+import subprocess
+import wave
+from pathlib import Path
+
+import pandas as pd
+import pycountry
+from omegaconf import DictConfig
+from tqdm import tqdm
+
+db_to_excel_metadata_names = {
+    "name": "name",
+    "email": "mail",
+    "age": "age",
+    "gender": "gender",
+    "dialect": "dialect",
+    "accent": "native_language",
+    "languages": "spoken_languages",
+    "zipcode_school": "zip_primary_school",
+    "zipcode_birth": "zip_grew_up",
+    "birth_place": "birthplace",
+    "education": "education",
+    "occupation": "occupation",
+}
+
+
+def make_speaker_metadata(cfg: DictConfig, raw_path: Path) -> pd.DataFrame:
+    """Make a speaker metadata dataframe from the raw data.
+
+    Args:
+        cfg (DictConfig):
+            The Hydra configuration object.
+        raw_path (Path):
+            The path to the raw data."""
+    # Load the metadata
+    metadata_path = raw_path / "metadata.xlsx"
+
+    # Make speaker-metadata file
+    speaker_metadata = pd.read_excel(metadata_path)
+
+    # Get all columns but not the column "Unnamed: 0"
+    speaker_metadata = speaker_metadata.loc[
+        :, ~speaker_metadata.columns.str.contains("^Unnamed")
+    ]
+
+    # Replace all nan values with empty strings
```

Comment:
Rather than stating what the code is doing, which here can be read simply from the code, it would be more valuable to explain why we prefer empty strings over NaN values here.